vendor/pacman/js/game.js

```diff
@@ -23,52 +23,81 @@ var TIME_FRUITS = 0;
 
 var HELP_DELAY = 1500;
 var HELP_TIMER = -1;
-			
-function blinkHelp() { 
-	if ( $('.help-button').attr("class").indexOf("yo") > -1 ) { 
+
+let gameOver = new Event("gameOver");
+
+window.onload = function() {
+   document.getElementById("panel").addEventListener("gameOver", scorePopUp);
+}
+
+function scorePopUp() {
+	var name = prompt("Please enter your name:", "anonymous");
+	// don't post if person cancels prompt or doesn't enter name value
+	if (name == null || name == "") {
+		SCORE = 0;
+		return;
+	}
+	var score = SCORE;
+	console.log(score);
```

Comment:
```suggestion
```
You might want to remove this console logging here and in `gameover` for cleanliness or else make it more clear with 
```js 
console.log(`Score: ${score}`);
```